File: Homework/mario.c
#include <stdio.h>
#include <cs50.h>


int main()
{
    int height, lines, space;
    
    do {
    printf("Enter the height: ");           // loop for entering integer data from 0 to 23
     height = GetInt();
    }
    while (height < 0 || height > 23);
    
   for (lines = 0; lines < height; lines++){         //loop for drawing lines
        for (space = height - lines - 1; space > 0; space--){ //   loop to format  "#" to the right edge
            printf(" ");
        }
        for (int i = 1; i <= (lines + 1); i++){
            printf("#");
        }
        printf("  "); /// gap between piramids
                
        for (int q = 0; q <= lines; q++){  //loop for printing the second piramid
                printf ("#");
        }
        printf("\n");
    }
    return 0;
}

File: pset6/mario.py
#!/usr/bin/env python3
import cs50

def main():
    n = get_height()  # gettint the height of the triangle
    output(n)    # printing the triangle
    
def get_height():

    while True:
        print("Enter the height: ", end="")
        n = cs50.get_int()
        if n >= 1 and n <= 23:
            break
    
    return n
    

def output(n):
    
    for line in range(n):     #height of the triangle
        for space in range(n-line-1):  #spaces for the printing the left triangle
            print(" ", end="")
            
        for tag in range(line+1):           #printing the left triangle
            print("#", end="")
            
        print("  ", end="")                 #gap between triangles
        
        for tag in range(line+1):           #printing the right triangle
            print("#", end="")
            
        print()

if __name__ == "__main__":
    main()